### ISSUE TITLE
Confusió entre imperatiu i present (i altres formes verbals)

### ISSUE THREAD

**Issue by jordimas** [apertium/apertium-spa-cat#19]
En els textos administratius hi ha bastants textos amb imperatius (p. ex. a formularis) que no traduïm bé la forma verbal.

Alguns exemples:

_En el cas de tenir reconegut un grau de discapacitat, indiqueu el percentatge:
En el caso de tener reconocido un grado de discapacidad, **indicáis** el porcentaje:
En el caso de tener reconocido un grado de discapacidad, **indique** el porcentaje:

Un imprès mitjançant el qual s’invita una o diverses persones a assistir a algun acte. També podeu invitar amb una carta (vegeu el model més amunt).
Un impreso mediante el cual se invita una o varias personas a asistir a algún acto. También podéis invitar con una carta (**veáis** el modelo más arriba).
Un impreso mediante el cual se invita una o varias personas a asistir a algún acto. También podéis invitar con una carta (**vean** el modelo más arriba).

SOL·LICITUD DE BAIXA (Indiqueu el motiu)
SOLICITUD DE BAJA (**Indicáis** el motivo)
SOLICITUD DE BAJA (**Indique** el motivo)_

Problema: en català indiqueu pot ser present d’indicatiu (indicáis) subjectiu (indiquéis) o bé imperatiu (indique) i cal desambiguar la forma correcta.

**Comment by jaumeortola** [comment]
Hi ha el canvi indicatiu/imperatiu, però també el canvi de tractament (vós en català, usted en castellà). No hi veig solucions fàcils. Potser convindria fer un recull dels verbs més usuals que es troben en aquestes situacions: indiqueu, vegeu, marqueu...

**Comment by MarcRiera** [comment]
L'altre dia vam comentar que potser podríem preferir l'imperatiu en alguns casos molt específics, com aquests verbs més freqüents quan es troben entre parèntesi i després d'una coma (si abans hi ha un condicional).

**Comment by jordimas** [comment]
@ja

> Hi ha el canvi indicatiu/imperatiu, però també el canvi de tractament (vós en català, usted en castellà). No hi veig solucions fàcils. Potser convindria fer un recull dels verbs més usuals que es troben en aquestes situacions: indiqueu, vegeu, marqueu...

Sí, els verbs que he vist fins ara són: indiqueu, marqueu, vegeu i truqueu

Si podem fer una solució per aquests, i després, si surten més es poden anar ampliant.

Amb un petit programa Python vaig mirar quan verbs els imperatius coincidia amb altres formes verbals i me'n sortien 8957 verbs.  Sense filar molt prim, hi ha molts, i jo només faria una cosa molt quirurjica pels verbs més comuns.
